src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -22,19 +33,48 @@ const CourseEnrollments = ({ children }) => {
     setShowMarkCourseCompleteSuccess,
     setShowMoveToInProgressCourseSuccess,
   } = useContext(CourseEnrollmentsContext);
+  const {
+    enterpriseConfig: {
+      slug,
+    },
+  } = useContext(AppContext);
+  const {
+    redeemableLearnerCreditPolicies,
+  } = useContext(UserSubsidyContext);
+
+  const [assignments, setAssignments] = useState([]);
+  useEffect(() => {
+    const data = redeemableLearnerCreditPolicies?.flatMap(item => item?.learnerContentAssignments || []);
+    setAssignments(data);
+  }, [redeemableLearnerCreditPolicies]);
+
+  const allocatedAssignments = assignments?.filter((assignment) => assignment?.state === ASSIGNMENT_TYPES.allocated
```

Comment:
I think we should rename `allocatedAssignments` since it represents cancelled assignments as well.